src/kafka-topics/view/view.controller.js

```diff
@@ -326,7 +326,7 @@ angularAPP.controller('ViewTopicCtrl', function ($scope, $rootScope, $filter, $r
     // If we don't know we need to guess by trying Avro -> JSon -> Binary
     var dataPromiseAvro = KafkaRestProxyFactory.consumeKafkaRest("avro", $scope.topicName);
     dataPromiseAvro.then(function (allData) {
-      if (JSON.stringify(allData).indexOf("error_code") > 0) {
+      if (JSON.stringify(allData).indexOf("error") > 0) { // JSON.stringify(allData).indexOf("error_code") returns -1
```

Comment:
something weird goes here but this code will be refactored. Will keep this for now in your new branch.